extensions/cloudfiles-extensions/pom.xml

```diff
@@ -0,0 +1,186 @@
+			<artifactId>rackspace-clouddatabases-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudqueues-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudloadbalancers-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-clouddns-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cdn-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudnetworks-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<!-- Rackspace UK dependencies -->
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudservers-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudfiles-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudblockstorage-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-clouddatabases-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudqueues-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-autoscale-us</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-cloudloadbalancers-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.provider</groupId>
+			<artifactId>rackspace-clouddns-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cdn-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.jclouds.labs</groupId>
+			<artifactId>rackspace-cloudnetworks-uk</artifactId>
+			<version>${jclouds.version}</version>
+		</dependency>
+		<!-- 3rd party dependencies -->
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-classic</artifactId>
+			<version>1.0.13</version>
```

Comment:
From the comment `<!-- jclouds dependencies -->` I just copied all the dependencies from this [pom](https://github.com/jclouds/jclouds-examples/blob/master/rackspace/pom.xml) which is the one cited in the [getting started guide of Rackspace cloud](https://jclouds.apache.org/guides/rackspace/).